### DISCUSSION TITLE
RSSI Entity (Wifi Signal Strength) for Sunton ESP32-3248S035C

### DISCUSSION THREAD

**Discussion by wilwizard1979** [HASwitchPlate/openHASP#481] [Q&A]
Hi, is there any way to get an RSSI (wifi signal strength) entity from Sunton displays?
After discovery, I added this device and this is what I get available:
![image](https://user-images.githubusercontent.com/108979356/236928043-a523eeac-d100-4796-959f-44e6cedfc14d.png)

Thanks in advance!

**Comment by wasn-eu** [comment]
that would be great.
i use 2 sunton display for my energy management and solar system

**Reply by htvekov** [reply]
The RSSI value is already in HA as one of the openHASP device attributes

![image](https://github.com/HASwitchPlate/openHASP/assets/45469813/1f004607-440a-40ad-8b08-94425bc5ee6b)

**Reply by zagi988** [reply]
It is in attributes but should be an entity, not an attribute.

**Reply by amanaplan** [reply]
If you absolutely need it as its own entity, then create a template helper sensor in the HA UI with:

`{{state_attr('openhasp.sunton_01', 'rssi')}}`
